app/layout.tsx

```diff
@@ -26,12 +26,12 @@ export default function RootLayout({
 }) {
   return (
     <html id="html" lang="en">
-      <body className={ inter.className }>
```

Comment:
looks like this got un-linted. Maybe you have an auto-format on save?
try this:
`ctrl + shift + p` --> open settings, not the json one.
--> search "format".
--> you should see `Editor: format on save mode`
select --> "modificationsIfAvailable" in the dropdown
```suggestion
      <body className={ inter.className }>
```